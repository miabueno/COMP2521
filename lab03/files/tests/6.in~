8
13
55
2
34
21
5
1
3
