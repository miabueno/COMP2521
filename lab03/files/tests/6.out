Tree:
1
 \
  2
   \
    3
     \
      5
       \
        8
         \
         13
           \
           21
             \
             34
               \
               55

#nodes = 9,  #leaves = 1
Infix  : 1 2 3 5 8 13 21 34 55 
Prefix : 1 2 3 5 8 13 21 34 55 
Postfix: 55 34 21 13 8 5 3 2 1 
ByLevel: 1 2 3 5 8 13 21 34 55 
